### ISSUE TITLE
G1000 - DA62 blank screens

### ISSUE THREAD

**Issue by LazyBoot** [Working-Title-MSFS-Mods/fspackages#215]
The G1000 in the DA62 only has all screens just completely black

using G1000-0.3.1

**Comment by Newerty** [comment]
I have same problem on G36 and G58 using 0.3.1

**Comment by LazyBoot** [comment]
Same issue on the SR22

**Comment by kaosfere** [comment]
I've been unable to reproduce black screens with 0.3.1 in these cases.   Can you please list what other mods you have installed?

**Comment by Newerty** [comment]
I have installed mostly liveries and sceneries. Besides that i have installed 'PHP-NoTooltips-Fix' and 'unreal-weather-live-metar' - will try without them.

**Comment by LazyBoot** [comment]
I removed all the mods from my "community" folder other than "workingtitle-g1000" and I still get the issue

**Comment by Newerty** [comment]
I know the reason. Check if folders named 'asobo-vcockpits-something' (11 foldres) are up to date / unmodified. Those folders are in 'Official' folder (next to 'Community' folder). You can acquire them by reinstalling (re-downloading whole content) the game or ask friend to send you them (its about 8mb).

**Comment by CkovMk** [comment]
I had the issue recently in DA62, but not in DA40NG. So I checked their code and find out G1000 system draw 40W + 30W of power in DA62, but less power in DA40NG(I forgot the exact numbers lol). I assume it is the G1000 that brought the whole electrical system down. A possible fix on this issue may be adjust power consumption of G1000 in DA62 system.cfg file, [ELECTRICAl] section.
I fixed this issue in my sim, but I have no clue whether it will fix yours, given some of you have issues also in SR22.
